Autopilot/AttitudeManager/FixedWing.c

```diff
@@ -0,0 +1,248 @@
+    //If commands come from the RC Controller
+    else setRollRateSetpoint(getRollRateInput(ROLL_RC_SOURCE));
+
+    //If commands come from the autopilot
+    if (getControlPermission(PITCH_CONTROL_TYPE, ANGLE_CONTROL,PITCH_CONTROL_TYPE_SHIFT)){
+        setPitchRateSetpoint(pitchAngleControl(getPitchAngleSetpoint(), -getPitch()));   //Keep a steady Pitch Angle
+        setPitchRateSetpoint(coordinatedTurn(getPitchRateSetpoint(), getRoll()));       //Apply Coordinated Turn
+    }
+    //If commands come from the ground station
+    else if (getControlPermission(PITCH_CONTROL_SOURCE, PITCH_GS_SOURCE,PITCH_CONTROL_SOURCE_SHIFT)){
+        setPitchRateSetpoint(getPitchRateInput(PITCH_GS_SOURCE));                         //Keep a steady Pitch Angle
+        setPitchRateSetpoint(coordinatedTurn(getPitchRateSetpoint(), getRoll()));       //Apply Coordinated Turn
+    }
+    //If commands come from the RC Controller
+    else{
+        setPitchRateSetpoint(getPitchRateInput(PITCH_RC_SOURCE));                         //Keep a steady Pitch Angle
+        setPitchRateSetpoint(coordinatedTurn(getPitchRateSetpoint(), getRoll()));       //Apply Coordinated Turn
+    }
+    
+    //If commands come from the ground station
+    if (getControlPermission(FLAP_CONTROL_SOURCE, FLAP_GS_SOURCE ,FLAP_CONTROL_SOURCE_SHIFT)){setFlapSetpoint(getFlapInput(FLAP_GS_SOURCE));}
+    //If commands come from the RC Controller
+    else if (getControlPermission(FLAP_CONTROL_SOURCE,FLAP_RC_SOURCE, FLAP_CONTROL_SOURCE_SHIFT)){setFlapSetpoint(getFlapInput(FLAP_RC_SOURCE));}
+
+    //If commands come from the ground station
+    if (getControlPermission(THROTTLE_CONTROL_SOURCE, THROTTLE_GS_SOURCE ,THROTTLE_CONTROL_SOURCE_SHIFT)){setThrottleSetpoint(getThrottleInput(THROTTLE_GS_SOURCE));}
+    //If commands come from the RC Controller
+    else if (getControlPermission(THROTTLE_CONTROL_SOURCE,THROTTLE_RC_SOURCE, THROTTLE_CONTROL_SOURCE_SHIFT)){setThrottleSetpoint(getThrottleInput(THROTTLE_RC_SOURCE));}
+
+    control_Roll = rollRateControl((float)getRollRateSetpoint(), -getRollRate());
+    control_Pitch = pitchRateControl((float)getPitchRateSetpoint(), -getPitchRate());
+    control_Yaw = yawRateControl((float)getYawRateSetpoint(), -getYawRate());
+    control_Throttle = getThrottleSetpoint();
+    
+    outputSignal[FLAP_OUT_CHANNEL - 1] = getFlapSetpoint(); // don't need to mix the flaps
+
+    //Mixing!
+    outputMixing(outputSignal, &control_Roll, &control_Pitch, &control_Throttle, &control_Yaw);
+    
+    //Error Checking
+    checkLimits(outputSignal);
+    //Then Output
+    unsigned int cameraCounter = 0; //TODO: TEMPORARY, STORE TIME NOT COUNTER (OR BOTH) IMPLEMENT THIS A BETTER WAY
+    cameraPollingRuntime(getLatitude(), getLongitude(), getTime(), &cameraCounter, getRoll(), getPitch());
+    cameraGimbalStabilization(getRoll());
```

Comment:
you can remove the files relating to the camera stuff as well (in a seperate PR)